.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@vickywane what is all this .idea stuff? Does it belong to .gitignore?

Reply:
It has been added in the `.gitignore`. They were auto-generated when I switched to using Webstorm IDE